cimetrics/env.py

```diff
@@ -28,8 +34,8 @@ def __init__(self) -> None:
                     self.cfg = {}
         else:
             print(
-                f"{self.CONFIG_FILE} does not exist at the root of your repo."
-                " Your metrics will not be recorded."
+                f"{self.CONFIG_FILE} does not exist at the root of the repo."
```

Comment:
I think we should print the full path here too "No configuration found at {path}, metrics will not be recorded".